### DISCUSSION TITLE
Error in PheKnowLator/notebooks/tutorials/entity_search.ipynb

### DISCUSSION THREAD

**Discussion by kangyh9659** [callahantiff/PheKnowLator#129] [Q&A] [answered]
Hello. I'm sorry but I'm trying to run the code from the entity_search.ipynb but I got stucked by some error. The error is:

NameError: name 'processes_ancestor_path_list' is not defined

This error comes from the following code:

`# examine the node's ancestors
prefix = 'CHEBI'
chebi6503_anc_dict = processes_ancestor_path_list(nx_ancestor_search(kg, node.copy(), prefix))
chebi6503_ancestors = format_path_ancestors(chebi6503_anc_dict, node_data_dict)`

To run this code, what I only did was to git clone this PheKnowLator. Probably, I need to install some other stuff?

If you could help me how to solve this problem, I would appreciate so much.
Thank you. Have a nice day.

**Comment by callahantiff** [comment, marked as answer]
Thanks @kangyh9659 -- so sorry about that! It looks like two utility functions were missing from the last PR. I have added them back via PR #130. Once the tests are done running. I will merge the PR and you should be able to update your cloned version to grab these changes.

For verification, the two functions (i.e., `nx_ancestor_search` and `processes_ancestor_path_list`) were being added to `kg-utils.py`.

Hopefully this fixes things, but if you find other problems let me know!

Thanks for using PheKnowLator! 😄

**Reply by kangyh9659** [reply]
Thank you for fixxing it. Hope you have a wonderful weekend!

**Reply by callahantiff** [reply]
No problem, hopefully there are no other snags. Please let me know if you run into anything (apologies in advance).

I hope you have a great weekend as well!
